### PR TITLE
Fix the layout to fill the entire screen

### PR DESCRIPTION
**Pull request by txsmith**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "carbon-elements": "polymerelements/carbon-elements#^0.1.0",
     "iron-elements": "polymerelements/iron-elements#^1.0.10",
     "app-layout": "polymerlabs/app-layout#^0.0.15",
-    "carbon-route": "TimvdLippe/carbon-route#subroute-updating"
+    "carbon-route": "TimvdLippe/carbon-route#subroute-updating",
+    "paper-header-panel": "PolymerElements/paper-header-panel#^1.1.5"
```
Comment (TimvdLippe) [inline bower.json]: Already included in `paper-elements`

File: src/rite-app.html [hunk 2]
```diff
@@ -51,18 +52,26 @@
...
 
-    <app-header-layout>
-      <!-- main header -->
-      <app-header id="header" effects="waterfall">
-        <app-toolbar id="mainToolbar">
-          <div class="spacer">
-            <div class="logo">PReview Code</div>
-          </div>
-        </app-toolbar>
+    <paper-header-pane>
```
Comment (TimvdLippe) [inline src/rite-app.html]: `paper-header-panel`

**Comment by eanker** [comment]
No description no merge

File: src/rite-app.html [hunk 3]
```diff
@@ -51,18 +52,26 @@
...
+        flex-direction: column;
+        height: 100vh;
+      }
+      .paper-header {
+        flex: 0;
+      }
+      iron-pages {
+        flex: 1;
+        display: flex;
+        flex-direction: column;
```
Comment (TimvdLippe) [inline src/rite-app.html]: Seems redundant.

**Pull request merged**